core/test/audits/uses-rel-preconnect-test.js

```diff
@@ -14,12 +14,14 @@ const mainResource = {
   url: 'https://www.example.com/',
   timing: {receiveHeadersEnd: 0.5},
   networkEndTime: 1000,
+  priority: 'High',
 };
 
 function buildArtifacts(networkRecords) {
   const trace = createTestTrace({
     timeOrigin: 0,
     largestContentfulPaint: 5000,
+    firstContentfulPaint: 5000,
```

Comment:
Can you add a test case that has LCP savings but not FCP savings? You should be able to achieve this by lowering this value to 4000 or something and then using a request that starts after FCP and ends before LCP in the test case.